pkgs/os-specific/linux/minimal-bootstrap/mes/sources.nix

```diff
@@ -2,21 +2,21 @@
 # Do not edit!
```

Comment:
It would be nice if ./gen-sources.sh fed this file through `nixpkgs-fmt` so that it's got a nicer diff. 🦺